### PR TITLE
BP-3092: New layout for expert view

### PR DESCRIPTION
**Pull request by bauer00**

### CODE REVIEW

File: packages/verification-vue-component/src/locales/en.json [hunk 1]
```diff
@@ -65,12 +65,16 @@
...
                 "registrationDate": "Issue Date & Time",
                 "registrationTransaction": "Registration Transaction",
                 "revocationTransaction": "Revocation Transaction",
                 "revocationDate": "Revocation Date & Time",
                 "smartContractAddress": "Smart Contract Address",
-                "signatureTransactions": "Signature transactions",
+                "signatureDetails": "Signature details",
+                "signatureLevel": "Signature Level",
+                "jurisdiction": "Jurisdiction",
+                "downloadSignature": "Download Signature",
```
Comment (bauer00) [inline packages/verification-vue-component/src/locales/en.json:77]: @stephalba is the english translation ok?

grafik

Comment (stephalba) [reply packages/verification-vue-component/src/locales/en.json:77]: I would bot capitalize the second word (even if some of them are in all caps).

Comment (bauer00) [reply packages/verification-vue-component/src/locales/en.json:77]: Ok makes sense, should I change that also for all the other labels in the `meta` section?

Comment (stephalba) [reply packages/verification-vue-component/src/locales/en.json:77]: Yes please.

File: packages/verification-vue-component/src/locales/de.json [hunk 2]
```diff
@@ -65,12 +65,16 @@
...
                 "registrationDate": "Ausstelldatum & -Zeit",
                 "registrationTransaction": "Ausstellungstransaktion",
                 "revocationTransaction": "Annullierungstransaktion",
                 "revocationDate": "Annullierungsdatum & -Zeit",
                 "smartContractAddress": "Smart Contract-Adresse",
-                "signatureTransactions": "Unterzeichnungstranskationen",
+                "signatureDetails": "Signature details",
+                "signatureLevel": "Signature level",
+                "jurisdiction": "Jurisdiction",
+                "downloadSignature": "Download signature",
```
Comment (bauer00) [inline packages/verification-vue-component/src/locales/de.json:77]: @stephalba could you please translate the changed labels to german?

File: packages/verification-vue-component/src/locales/fr.json [hunk 3]
```diff
@@ -65,12 +65,16 @@
...
                 "registrationDate": "Date & heure de l'émission",
                 "registrationTransaction": "Transaction d'énregistrement",
                 "revocationTransaction": "Transaction de révocation",
                 "revocationDate": "Date & heure de la révocation",
                 "smartContractAddress": "Adresse du Smart Contract",
-                "signatureTransactions": "Transactions des signatures",
+                "signatureDetails": "Signature details",
+                "signatureLevel": "Signature level",
+                "jurisdiction": "Jurisdiction",
+                "downloadSignature": "Download signature",
```
Comment (bauer00) [inline packages/verification-vue-component/src/locales/fr.json:77]: @emeka could you please translate the changes labels to french?

File: packages/verification-vue-component/src/locales/it.json [hunk 4]
```diff
@@ -65,12 +65,16 @@
...
                 "registrationDate": "Data e ora di emissione",
                 "registrationTransaction": "Transazione di emissione",
                 "revocationTransaction": "Transazione di annullamento",
                 "revocationDate": "Data e ora di annullamento",
                 "smartContractAddress": "Indirizzo del Smart Contract",
-                "signatureTransactions": "Transazioni di firma",
+                "signatureDetails": "Signature details",
+                "signatureLevel": "Signature level",
+                "jurisdiction": "Jurisdiction",
+                "downloadSignature": "Download signature",
```
Comment (bauer00) [inline packages/verification-vue-component/src/locales/it.json:77]: @ianaz could you please translate the changed labels to italian?

Comment (ianaz) [reply packages/verification-vue-component/src/locales/it.json]: Sure :) 
d4fe284

File: packages/verification-vue-component/src/locales/en.json (hunk 1 above)
Comment (bauer00) [reply packages/verification-vue-component/src/locales/en.json]: Done

File: packages/verification-vue-component/src/locales/it.json (hunk 4 above)
Comment (bauer00) [reply packages/verification-vue-component/src/locales/it.json]: Thanks 🙏

File: packages/verification-vue-component/src/locales/de.json (hunk 2 above)
Comment (stephalba) [reply packages/verification-vue-component/src/locales/de.json]: Done

Comment (bauer00) [reply packages/verification-vue-component/src/locales/de.json]: Thanks

**Review by alnicoli — APPROVED** [review]

**Pull request merged**